### PR TITLE
Adds compass rose

### PR DESCRIPTION
**Pull request by RafaelCosman**
To help the user stay oriented.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -228,9 +228,6 @@ bugg = 1000;
     geometry.vertices.push( new THREE.Vertex( new THREE.Vector3( (axisMax+1) * 50, 0, 0 ) ) );
     linesMaterial = new THREE.LineColorMaterial( 0x000000, 0.2 );
     
-
```
Comment (RafaelCosman) [inline js/index.js]: Deleted unnecessary whitespace

File: js/index.js [hunk 2]
```diff
@@ -273,13 +270,42 @@ bugg = 1000;
     //might be to figure out what is in front of the camera
     ray = new THREE.Ray( camera.position, null );
     
+
+    //Add a compass rose to help the user stay oriented
+    var compassRoseOffsetFromCenter = 630;
+    compassRoseAlpha = .5;
+
+    compassRose = new THREE.Mesh( new Cube(10, 10, 100), new THREE.MeshColorFillMaterial( 0xFF0000, compassRoseAlpha) );
```
Comment (RafaelCosman) [inline js/index.js]: The compass rose is made out of three 'cubes'.

File: js/index.js [hunk 3]
```diff
@@ -273,13 +270,42 @@ bugg = 1000;
...
+    scene.addObject( compassRose );
+
+
     // this is the cursor that shows where you are going to create a cube
     brush = new THREE.Mesh( cube, new THREE.MeshColorFillMaterial( colors[ color ], 0.4 ) );
     //brush.position.y = brushY;
     setObjPosition(brush, cursor);
     brush.overdraw = true;
     scene.addObject( brush );
+
```
Comment (RafaelCosman) [inline js/index.js]: Added this spacing to make the code here clearer

**Comment by RafaelCosman** [comment]
Please take a look at this, @danx0r @brianjoseff.

**Comment by RafaelCosman** [comment]
@veered When we finish updating to the new version of THREE.js, this compass rose will unfortunately be one more thing that we'll have to update. But it's not making the problem much worse thankfully : P. Thank you for getting that update started!

**Comment by danx0r** [comment]
need to adjust for size -- try file:///home/dbm/BusyBoxes/index.html?hash=Y0&size=40
or
file:///home/dbm/BusyBoxes/index.html?hash=Y0&size=10

**Comment by RafaelCosman** [comment]
@danx0r Thanks for pointing that out. Let me know if I've fixed it.
